starter-code/iteration-1/main.ts

```diff
@@ -1,12 +1,36 @@
 // Iteration 1
 // 1. Create an array of strings
+const taskList:Array<string> = [];
 
-// 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+// class toDoList {
```

Comment:
In general, this works but we are practicing typescript and ES6 so practicing classes would be nice. So we would have something like this: